TaskList/app/views/tasks/index.html.erb

```diff
@@ -1,7 +1,39 @@
 <h1>Task List</h1>
+<div id='divider'>-----------------</div>
```

Comment:
Depending on the visual style you're going for, it might be easier to use the `<hr>` tag for this instead of a `<div>`.